src/python/pants/option/option_value_container.py

```diff
@@ -20,25 +24,28 @@ class OptionValueContainer:
      See ranked_value.py for more details.
   """
 
-  def __init__(self):
-    self._value_map = {}  # key -> either raw value or RankedValue wrapping the raw value.
```

Comment:
This was a stale comment. All the code assumes that its a `Dict[Key, RankedValue]` and it would fail at runtime were it `Dict[Key, Union[Value, RankedValue]]`